### PR TITLE
fix: Update schema to detect duplicates

### PR DESCRIPTION
**Pull request by ermshiperete**
This change updates the schema as suggested in #17 to detect duplicates in tags. It also adds a unit tests to detect duplicates that appear in multiple entries, as suggested in #16.

This change does not update the data and thus will cause failing tests.

### CODE REVIEW

**Comment by mhosken** [comment]
Could you remove the need for unique items in latnnames, as described in #17 and the docs. Also I have just added an identical test to the tests (in test_supplemental.py) so that's probably not needed. But thank you for creating a PR and working it through.

File: source/langtags_schema.json [hunk 1]
```diff
@@ -76,11 +81,13 @@
...
           "type": "array",
-          "items": { "type": "string" }
+          "items": { "type": "string" },
+          "uniqueItems": true
         },
         "latnnames": {
           "type": "array",
-          "items": { "type": "string" }
+          "items": { "type": "string" },
+          "uniqueItems": true
```
Comment (mhosken) [inline source/langtags_schema.json]: This need not be unique. The key is that each entry aligns with a corresponding entry in the localnames array.
